### ISSUE TITLE
[BUG] Date format issue

### ISSUE THREAD

**Issue by ashishpadave** [RosaeNLG/rosaenlg#47]
**Describe the bug**
`p
  | #[+value( new Date('1980-04-14') , {dateFormat:'MMMM d, Y hh:mm OOOO'})]
`
Outputs
`<p>April 14, 1980 05: 30  GMT+05: 30</p>`
There is an extra space between the hr and the mins and also in the Timezone
**To Reproduce**
Steps to reproduce the behavior

**Expected behavior**
There should be no space 

**Code**
![rosaebug](https://user-images.githubusercontent.com/535837/106281004-a508c880-6264-11eb-8bc8-8a0914cbb5a1.png)

**Context:**
 - RosaeNLG version: [e.g. 1.19.1, 1.20.2, etc.]
 - Language: [e.g. en_US, fr_FR, etc.]
 - Packaging: [node.js, browser, Java]

**Additional context**

**Comment by ludans** [comment]
Thanks @ashishpadave
I confirm the issue.

**Cause**

 The ":" is mistakenly considered as a colon punctuation and RosaeNLG adds a space after the colon.

**Workaround**

As a workaround, please use the `protect` tag:
```
protect
  | #[+value(new Date('1980-04-14') , {dateFormat:'MMMM d, Y hh:mm OOOO'})]
```
will output `April 14, 1980 02:00 GMT+02:00`

<img width="684" alt="protect" src="https://user-images.githubusercontent.com/20767031/106307030-1d6c8b00-625f-11eb-9721-ad18dcfeb829.PNG">

**Root cause**

"protect" should be enabled by default when outputting the value of a date (i.e. it should not be transformed).
There are edge cases on French that I must check.
Expect a solution without using "protect" in the next version.

**Comment by ashishpadave** [comment]
Thank you @ludans .

**Comment by ludans** [comment]
corrected in 2.1.4, just published
